src/components/application_manager/test/app_launch/app_launch_data_json_test.cc

```diff
@@ -68,7 +68,7 @@ class AppLaunchDataJsonTest : public ::testing::Test {
   virtual void SetUp() {
     ::file_system::DeleteFile(kAppStorageFile);
     test_last_state_ = std::auto_ptr<resumption::LastState>(
```

Comment:
@LuxoftAKutsan You right, changed this class to handle LastState by interface in 2d675aa.